### PR TITLE
Convention for static/spa locations in the packages

### PR DESCRIPTION
**Pull request by ranatrk**
Refactor the following packages for the convention used along with https://github.com/threefoldtech/jumpscaleX_core/pull/161:

- ffbrowser
- alerta
- interface
- myjobs
- packages_manager

Issue: https://github.com/threefoldtech/jumpscaleX_threebot/issues/190

To minimize package.py code, the location creation is done in ThreebotPackage.py based on the dir structure of the packages being loaded where they contain /html directory for static apps or they contain /frontend fir for spa apps

### CODE REVIEW

File: ThreeBotPackages/zerobot/alerta/package.py [hunk 1]
```diff
@@ -11,21 +11,3 @@ def start(self):
...
-            website_location.path_url = "/alerta"
-            website_location.use_jumpscale_weblibs = False
-            fullpath = j.sal.fs.joinPaths(self.package_root, "html/")
-            website_location.path_location = fullpath
-
-            locations.configure()
-            website.configure()
-
-            # setup alert handler to intercept errors
-            j.tools.alerthandler.setup()
```
Comment (abom) [inline ThreeBotPackages/zerobot/alerta/package.py]: This call should've been outside the loop, we need to keep it here.

```
            # setup alert handler to intercept errors	
            j.tools.alerthandler.setup()
```

File: ThreeBotPackages/zerobot/packages_manager/package.py [hunk 2]
```diff
@@ -17,7 +17,7 @@ def start(self):
             website_location.name = "packages"
             website_location.path_url = "/packages"
             website_location.use_jumpscale_weblibs = True
-            fullpath = j.sal.fs.joinPaths(self.package_root, "html/")
```
Comment (abom) [inline ThreeBotPackages/zerobot/packages_manager/package.py]: I think we can remove the whole for loop block, as it should be created manually now.

For weblibs, i think the whole `use_jumpscale_weblibs` options can be removed, as it is always available at `/weblibs` (added in webinterface package by default), any package that uses this option need to be updated to use `/weblibs` directly.

Comment (ranatrk) [reply ThreeBotPackages/zerobot/packages_manager/package.py]: so `use_jumpscale_weblibs` is no longer needed even in ThreebotPackage location adding?

Comment (abom) [reply ThreeBotPackages/zerobot/packages_manager/package.py]: Yes, but if a package expects it to be exposed elsewhere, the package should update its code (e.g. frontend) to use `/weblibs`.

**Comment by xmonader** [comment]
Something doesn't seem right, 16K files changes?

**Comment by xmonader** [comment]
Sorry, I redid it and pushed

**Pull request closed without merging**